src/main/java/graphql/util/Anonymizer.java

```diff
@@ -260,16 +259,6 @@ public TraversalControl visitGraphQLFieldDefinition(GraphQLFieldDefinition graph
 
             @Override
             public TraversalControl visitGraphQLDirective(GraphQLDirective graphQLDirective, TraverserContext<GraphQLSchemaElement> context) {
-                if (Directives.DEPRECATED_DIRECTIVE_DEFINITION.getName().equals(graphQLDirective.getName())) {
-                    GraphQLArgument reason = newArgument().name("reason")
-                            .type(Scalars.GraphQLString)
-                            .clearValue().build();
-                    GraphQLDirective newElement = graphQLDirective.transform(builder -> {
-                        builder.description(null).argument(reason);
-                    });
-                    changeNode(context, newElement);
-                    return TraversalControl.ABORT;
-                }
```

Comment:
I am guessing the Anonymizer pre-dates the split of GraphQLDirective and GraphQLAppliedDirective

Now that applied directives are separated, there shouldn't be any transformation in this `GraphQLDirective` visitor method. The deprecated directive as defined in the schema should be unchanged, as any other built-in directive in the next lines

```
if (DirectiveInfo.isGraphqlSpecifiedDirective(graphQLDirective.getName())) {
    return TraversalControl.ABORT;
}
```